specification/automation/resource-manager/Microsoft.Automation/2015-10-31/definitions.json

```diff
@@ -2954,6 +2954,21 @@
       },
       "description": "The runbook property associated with the entity."
     },
+    "scheduleFrequency": {
+      "type": "string",
+      "description": "Gets or sets the frequency of the schedule.",
```

Comment:
In last review Azure team were against using gets and sets.  Can you change the description

Reply:
updated all! Thanks!